### PR TITLE
implement initial GlobeNewswire parser

### PR DESCRIPTION
**Pull request by petrjasek**
SDCP-123

### CODE REVIEW

File: server/cp/ingest/parser/globenewswire.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        item['slugline'] = 'GNW-{lang}-{time}--{symbols}'.format(
+            lang=item['language'],
+            time=meta.find(self.qname('contentCreated')).text[17:19],
+            symbols='-'.join(self._get_stock_symbols(tree)),
+        )
+
+        keywords = meta.findall(self.qname('keyword'))
+        item['keywords'] = [k.text for k in keywords if k.text and k.get('role') == KEYWORD_ROLE]
+
+        item['description_text'] = DESCRIPTION
```
Comment (ride90) [inline server/cp/ingest/parser/globenewswire.py:43]: Does it make sense to translate a description?

File: server/cp/ingest/parser/globenewswire.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        item['body_footer'] = BODY_FOOTER[item['language']]
```
Comment (ride90) [inline server/cp/ingest/parser/globenewswire.py:44]: `item['language']` can be only `en` and `fr`, right? Otherwise, it's better to return a default.

Comment (petrjasek) [reply server/cp/ingest/parser/globenewswire.py:44]: that's right

File: server/cp/ingest/parser/globenewswire.py (hunk 1 above)
Comment (petrjasek) [reply server/cp/ingest/parser/globenewswire.py:43]: I think it does, but don't have such specs

**Pull request merged**